diagnostics/model_test/mnist_cnn_with_inputs.json

```diff
@@ -0,0 +1,12 @@
+{
+    "model_name": "mnist_cnn",
+    "reference_backend": "CPU",
+    "testing_backend": "NGRAPH_CPU",
+    "graph_location": "/nfs/site/home/mingshan/frozen_graphs/frozen_mnist.pb",
```

Comment:
Once, load checkpoint PR is merged this parameter need to be changed to "pb_graph_location"